velox/row/UnsafeRowDeserializers.h

```diff
@@ -528,6 +529,23 @@ struct UnsafeRowPrimitiveBatchDeserializer {
   static StringView deserializeStringView(std::string_view data) {
     return StringView(data.data(), data.size());
   }
+
+  static int128_t deserializeLongDecimal(std::string_view data) {
+    const uint8_t* bytesValue = reinterpret_cast<const uint8_t*>(data.data());
+    int length = data.size();
+    uint8_t bytesValue2[32]{};
+    for (int k = length - 1; k >= 0; k--) {
+      bytesValue2[length - 1 - k] = bytesValue[k];
```

Comment:
What does this do?

Reply:
Convert big endian to little endian, I will add comment to explain where it come from

Reply:
I see. Can we use folly::Endian::big or folly::Endian::little? See example in UnsafeRowVectorSerde::deserialize.

Reply:
folly endian can only read number. Here I need to read char* by little endian, I receive error:
```
[build] /usr/local/include/folly/lang/Bits.h:306:44:   required from ‘static T folly::Endian::big(T) [with T = const char*]’
[build] .././velox/row/UnsafeRowDeserializers.h:537:74:   required from here
[build] /usr/local/include/folly/lang/Bits.h:257:69: error: static assertion failed: template type parameter must be non-bool integral or floating point
[build]   257 |       (std::is_integral<T>::value && !std::is_same<T, bool>::value) ||
[build]       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
[build]   258 |           std::is_floating_point<T>::value,
```